samples/csharp_dotnetcore/53.teams-messaging-extensions-action-preview/README.md

```diff
@@ -2,7 +2,7 @@
 
 Bot Framework v4 Teams Messaging Extension Action Preview sample.
 
-This Messaging Extension has been created using [Bot Framework](https://dev.botframework.com). It shows how to create a simple card based on parameters entered by the user from a Task Module.
+This Messaging Extension has been created using [Bot Framework](https://dev.botframework.com). It shows how to create a simple card based on parameters entered by the user from a Task Module.it also display the scenairo where a Bot sends messages on behalf of a User, attributing the message to that user can help with engagement and showcase a more natural interaction flow.
```

Comment:
Please remove this altogether - this is a minor feature compared to the overall purpose of the sample. If you want to enumerate the functionality in the sample to include this, we'd need to refactor this into like a bulleted list of things, where this was one of those things.